content/5_Plat/Bitsets.mdx

```diff
@@ -85,6 +136,63 @@ below).
 
 <Spoiler title="Full Solution">
 
+<LanguageSection>
+<PySection>
+
+```py
+# BeginCodeSnip{DSU}
+class DisjointSet:
```

Comment:
why doesn't the c++ sol use dsu but this one does

Reply:
What do you mean ? It uses DSU .
![image](https://github.com/cpinitiative/usaco-guide/assets/59292725/40486d3e-2496-4186-aef7-ceb54b023c32)

Reply:
I added the DSU codesnip to the C++ code